src/main/kotlin/org/rust/ide/formatter/impl/alignment.kt

```diff
@@ -0,0 +1,32 @@
+package org.rust.ide.formatter.impl
+
+import com.intellij.formatting.Alignment
+import com.intellij.openapi.util.Key
+import com.intellij.psi.tree.TokenSet
+import org.rust.ide.formatter.RustAlignmentStrategy
+import org.rust.ide.formatter.blocks.RustFmtBlock
+import org.rust.lang.core.psi.RustCompositeElementTypes.*
+
+/**
+ * Stores common alignment object for function declarations's parameters, return type & where clause.
+ */
+val PARAMETERS_ALIGNMENT: Key<Alignment> = Key.create("PARAMETERS_ALIGNMENT")
```

Comment:
Hm, I am still not sure that `UserDataHolder` is a superior approach to plain old fields :)